controllers/k8ssandracluster_controller.go

```diff
@@ -462,6 +472,21 @@ func (r *K8ssandraClusterReconciler) setStatusForStargate(kc *api.K8ssandraClust
 	return nil
 }
 
+func (r *K8ssandraClusterReconciler) ensureStargateAuthKeyspaceExists(
```

Comment:
We should only call `ensureStargateAuthKeyspaceExists` once, not once per Stargate instance. We can address this in #106.